app/helpers/organisation_helper.rb

```diff
@@ -2,4 +2,20 @@ module OrganisationHelper
   def child_organisations_count(organisation)
     organisation["works_with"].values.flatten.count
   end
+
+  def child_organisation_count(organisation)
+    if organisation.content_item.content_item_data["links"]["ordered_child_organisations"].count != 0
```

Comment:
you can use `.zero?` instead for example:

```ruby
unless organisation.content_item.content_item_data["links"]["ordered_child_organisations"].count.zero?
```

Reply:
I've had to do organisation.content_item.content_item_data["links"]["ordered_child_organisations"].presence as some organisations do not have this field in the content store to begin with so it is nil therefore doing a count throws an error